data-model/SDP-sample-records/ms-object_sample1.json

```diff
@@ -0,0 +1,85 @@
+{
+  "id": "ark:/21198/z1x07bdf",
+  "type": "Manuscript Object",
+  "shelfmark": "Sinai Syriac 2A",
+  "collection": "Old Collection",
+  "repository": "https://sinaimanuscripts.library.ucla.edu/place/1",
+  "form": "Codex",
+  "binding": {
+    "exists": true,
+    "type": "",
+    "note": ["Relatively recent; leather on wood showing signs of repair; metal studs (some missing) on front and back and along edges; remnants of clasps (four total, two for edge and one each at top and bottom)"],
+    "features": []
+  },
+  "extent": ["f. 1-180"],
+  "weight": "3788.3 g",
+  "dimensions": [{
+    "type": "object",
+    "value": "328 x 255 x 97 mm"
+  }],
+  "condition": {
+    "note": [],
+    "features": []
+  },
+  "provenance": [{
+    "type": "acquisition",
+    "note": ["Created and remained in Saint Catherine's Monastery"],
+    "features": [],
+    "source": []
+  }],
+  "paratexts": [],
```

Comment:
For the paratexts, persons, places, and dates fields.

If we had paratexts related to the production, use, ownership, or custodianship of the ms object as a whole we could include them here. Similarly the below fields would be for people, places, and dates attested in paratextual material associated with the ms object level -- or which are otherwise associated through indirect evidence with the ms object level.

This is maybe a larger question I'm coming up against also with the taxonomy of person, etc. types. We have both the object scope (is it related to the ms, the part, or the textual artifact) and the 'event' scope (is it about the production, use, ownership, or custodianship/conservation of the given unit)

Reply:
Paratext and Event become the base units. These are essentially mini-records embedded in the main object types (ms object, units, and textual artifacts). Give them a unique ID. Put in associated dates, persons, places, etc. here rather than in their separate fields.

Need a typology for paratexts and for events (production and provenance)